.github/workflows/docker-build.yml

```diff
@@ -134,4 +156,4 @@ jobs:
           -Djib.console=plain \
           -Djib.container.labels="$(echo "${{ steps.meta.outputs.labels }}" | tr '\n' ',' | sed 's/,$//')" \
           -Djib.to.tags="$(echo "${{ steps.meta.outputs.tags }}" | tr '\n' ',' | sed 's/,$//')"
-        docker push ${{ env.REGISTRY }}/${{ github.repository_owner }}/ort-server-${{ matrix.docker.image }} --all-tags
+        docker push ${{ env.REGISTRY }}/${{ github.repository_owner }}/ort-server-${{ matrix.docker.jibImage }} --all-tags
```

Comment:
Just curious: Why are you pushing the images with `docker push` instead of using `Jib`'s push functionality?

Reply:
I had some issues with using the `jib` task for pushing, but I don't remember the details. I think the problem was that when pushing multiple tags with `jib` they appeared as separate images in the GitHub packages UI. When pushing multiple tags with docker only one image with multiple tags is shown.